exchanges/binance/binance_test.go

```diff
@@ -2761,11 +2744,11 @@ func TestFetchExchangeLimits(t *testing.T) {
 	t.Parallel()
 	limits, err := b.FetchExchangeLimits(t.Context(), asset.Spot)
 	assert.NoError(t, err, "FetchExchangeLimits should not error")
-	assert.NotEmpty(t, limits, "Should get some limits back")
+	assert.NotEmpty(t, l, "Should get some limits back")
 
 	limits, err = b.FetchExchangeLimits(t.Context(), asset.Margin)
```

Comment:
```suggestion
	l, err = b.FetchExchangeLimits(t.Context(), asset.Margin)
```
Same issue with the intended var usage